float64[] sphere_center
float64 radius
float64 height
